policyengine_us/variables/gov/states/ct/tax/income/credits/ct_eitc.py

```diff
@@ -0,0 +1,26 @@
+from policyengine_us.model_api import *
+
+
+class ct_eitc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Connecticut Earned Income Tax Credit"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://portal.ct.gov/-/media/DRS/Forms/2022/Income/Schedule-CT-EITC_1222.pdf"
+    defined_for = StateCode.CT
+
+    def formula(tax_unit, period, parameters):
+        eitc = tax_unit("earned_income_tax_credit", period)
+        rate = parameters(
+            period
+        ).gov.states.ct.tax.income.credits.eitc.rate
+        filing_status = tax_unit("filing_status", period)
+        filing_statuses = filing_status.possible_values
+        separate_status = (filing_status == filing_statuses.SINGLE)
+        agi_separate = tax_unit("adjusted_gross_income", period)
+        #how do differentiate the two AGI's?
+        agi_joint = None
+        if separate_status:
```

Comment:
need to use a where( statement instead of the if statement - there are examples in the code of it

Reply:
Just so I understand, what functionality does the where statement give that would make it better than if statement?

Reply:
we can not use if statements in our model in general as it would loop and create breaks in the microsim

Reply:
Ok cool, and how do I differentiate between agi_separate and agi_joint

Reply:
you can use `adjusted_gross_income_person` to check for each person and `adjusted_gross_income` if they combine it in a joint return

Reply:
Thank you